### ISSUE TITLE
paypal.com

### ISSUE THREAD

**Issue by wanhose** [wanhose/cookie-dialog-monster#744]
#### Browser
Firefox (129.0)
#### Reason
Popup showed up
#### URL
https://www.paypal.com/paypalme/wanhose
#### Version
7.1.0

**Comment by dreirund** [comment]
The following cookie popup showed up once (opened the page in the background).

I could not reproduce it later, though.

---

![image](https://github.com/user-attachments/assets/7826c522-2dd5-4243-bd6f-4ca26c311fb4)

**Comment by wanhose** [comment]
Hey @dreirund,

I can say, sometimes, pages cache the extension scripts so could be that maybe first time you enter to a page it had the previous version of that script. It's crazy but I needed to do some hot refreshes of various pages to get the latest changes from time to time. So I recommend you, if you see this problem that you do a hot refresh pressing Ctrl + Shift + R in the page when you see the problem and if it persists... then we have a problem.

But I've entered several times, leaving it at the background and directly opening it and I'm not able to reproduce this issue 😅
